src/EnergyPlus/ChillerReformulatedEIR.hh

```diff
@@ -67,7 +67,7 @@ namespace ChillerReformulatedEIR {
 
     enum class PLR
     {
-        Unassigned,
+        Unassigned = -1,
```

Comment:
A few more notes as you are doing this pass. First, add `Num` at the end.

Reply:
To that point: assuming they also don't have `Unassigned` or `Invalid`, I should add those here?

Reply:
Yes. And that should be the only negative `enum` value. I know there are a few places where we have multiple negative `enum` values, those need to be flagged.